templates/cloud/thank-you.html

```diff
@@ -9,7 +9,7 @@
 {% block content %}
 {% include "shared/_thank_you.html" with thanks_context="enquiring about Cloud" %}
 
-	
+
 {% endblock content %}
 {% block footer_extra %}
 <!-- Google Code for Services Canonical &mdash; contact us Conversion Page -->
```

Comment:
I'm pretty sure this block of code for google ad services is exactly duplicated many times throughout the codebase. It could probably benefit from being an include.